### DISCUSSION TITLE
Card-layout based on screen-size

### DISCUSSION THREAD

**Discussion by anthonyma94** [Clooos/Bubble-Card#684] [Questions about config, custom styles and templates] [answered]
Right now I have 2 dashboards, one for desktop and one for mobile. They're identical with one having normal layouts and one having large 2 row layouts. Is it possible to consolidate the two dashboards into one while keeping normal layouts for mobile and 2 row layouts for desktop?

**Comment by MrBearPresident** [comment, marked as answer]
See #1094
